docset/winserver2019-ps/tls/Get-TlsCipherSuite.md

```diff
@@ -11,7 +11,7 @@ title: Get-TlsCipherSuite
 # Get-TlsCipherSuite
 
 ## SYNOPSIS
-Gets the list of cipher suites for TLS for a computer.
+Gets the cipher suites for TLS for a computer.
```

Comment:
Could this be rephrased to "Gets the list of TLS cipher suites available on the computer."?

Reply:
My thinking here is that the command returns a collection of objects. A list or table is the result of default formatting and not the command.. So the command gets the cypher suites. 

At least that is why I made the change.